apps/SageAccountWeb/src/style/_Register.scss

```diff
@@ -24,8 +24,8 @@
   }
   .thankYouContainer {
     background: #fefefe;
-    box-shadow: '0px 2px 6px rgba(53, 58, 63, 0.1)';
-    border-radius: '5px';
+    box-shadow: 0px 2px 6px rgba(53, 58, 63, 0.1);
+    border-radius: 5px;
```

Comment:
Fix box-shadow and border-radius:

main | feature
:---:|:---:
![main_register1](https://user-images.githubusercontent.com/26949006/229907867-7f03726c-9430-4d50-a841-70c0f749a689.png) | ![feature_register1](https://user-images.githubusercontent.com/26949006/229907881-2ccf8020-e250-4d5d-8724-f77ffc41e5b3.png)